releases.md

```diff
@@ -4,15 +4,15 @@ description: Home
 layout: docwithnav
 ---
 {% if site.archive %}
-- [latest](/)
+- [{{site.data.versions.first.title}} (latest)](/)
 - [nightly](/master)
 {%- for version in site.data.archives -%}
 {%- if version.first -%}
     {%- for v in version["legacy"] %}
-- [{{ v }}](/{{ v }})
+- [{{ v }}]({{ site.url }}/{{ v }})
```

Comment:
For now, I've disabled htmlproofer checking across versions. You can do so here with this code:
```suggestion
- [{{ v }}](/{{ v }}){: data-proofer-ignore=""}
```

In the future I may update the build machinery to build the entire site on each PR and validate links on it. If I do, I will remove the attribute.